datalake/src/test/java/com/sequenceiq/datalake/controller/sdx/SdxControllerTest.java

```diff
@@ -190,6 +197,21 @@ void refreshDatahubs() {
         verify(sdxService, times(1)).refreshDataHub(SDX_CLUSTER_NAME, null);
     }
 
+    @Test
+    void validateBackupStorage() {
+        SdxCluster sdxCluster = getValidSdxCluster();
+        sdxCluster.setClusterName(SDX_CLUSTER_NAME);
+        SdxBackupLocationValidationRequest sdxBackupLocationValidationRequest = new SdxBackupLocationValidationRequest(SDX_CLUSTER_NAME);
+        when(storageValidationService.validateBackupStorage(any())).thenReturn(null);
+        when(sdxService.getByNameInAccount(USER_CRN, SDX_CLUSTER_NAME)).thenReturn(sdxCluster);
+
+        ThreadBasedUserCrnProvider.doAs( USER_CRN, () -> sdxController.validateBackupStorage(sdxBackupLocationValidationRequest));
+
+        verify(sdxService, times(1)).getByNameInAccount
+                (USER_CRN, SDX_CLUSTER_NAME);
+        verify(storageValidationService, times(1)).validateBackupStorage(sdxCluster);
```

Comment:
As mentioned in a former comment, while this does not hurt, it is better to use a proper arg matching for `sdxCluster` in the `when()` invocation above. Especially since `validateBackupStorage()` has a non-`void` return type, so its output has a direct influence on the caller method (namely, being the result of `SdxController.validateBackupStorage()`).

Reply:
addressed